testsuite/features/secondary/srv_scc_user_credentials.feature

```diff
@@ -0,0 +1,43 @@
+# Copyright 2017-2021 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: SCC user credentials in the Setup Wizard
+
+
```

Comment:
Let's have some description.
The purpose of this feature is not the same that the feature in core, this feature aimes to test the manipulation of scc credentials while the other in core aimes to setup scc credentials.

Reply:
Agreed. That's a good idea indeed.